upb/pb/decoder.c

```diff
@@ -596,6 +596,7 @@ int32_t upb_pbdecoder_skipunknown(upb_pbdecoder *d, int32_t fieldnum,
     }
 
     if (d->top->groupnum >= 0) {
+      upb_sink_putunknown(&d->top->sink, d->checkpoint, d->ptr - d->checkpoint);
```

Comment:
Please remove the TODO above.

Also I think this code is incorrect for the case of unknown groups. But it might be a little complicated to fix right now. Maybe just add a TODO that the code needs to be fixed for unknown groups.

Reply:
done